ADR/0023-release-and-releaseplan-authorization.md

```diff
@@ -0,0 +1,55 @@
+# Release and ReleasePlan Authorization
+
+Date: 2023-04-17
+
+## Status
+
+Accepted
+
+## Context
+
+* It is imperative to know which associate has authorized a change to production in order to have accountability for changes.
```

Comment:
The term associate seems a little corporate. We can change this to an individual.

Are we limiting to a single individual, or is it possible to have multiple individuals?

Reply:
updated.